### PR TITLE
Document how to drop alert rules

### PR DESCRIPTION
**Pull request by corvus-ch**

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+= Remove alert rules
+
+[abstract]
+Somtimes an alert rule doesn't makes sense.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc:4]:
```suggestion
Sometimes an alert rule doesn't make any sense.
```

File: docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Be it that the rule is build on assumptions that don't apply.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc:5]:
```suggestion
Be it that the rule is built on assumptions that don't apply.
```

File: docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+[source,yaml]
+----
+parameters:
+  openshift4_monitoring:
+    alerts:
+      ignoreNames:
+        - MyAlertName
+----
++
+Do so on what ever hierarchy level is the most apropriate.
```
Comment (tobru) [inline docs/modules/ROOT/pages/how-tos/monitoring/remove_rules.adoc:30]:
```suggestion
Do so on whatever hierarchy level is the most appropriate.
```

**Review by tobru — CHANGES_REQUESTED** [review]
Besides some typos LGTM. We'll see in practice if this is sufficient or need more explanation.

**Review by ccremer — APPROVED** [review]
I would have found the same typos :)

**Review by tobru — APPROVED** [review]

**Pull request merged**